sahara.c

```diff
@@ -319,7 +316,10 @@ static ssize_t sahara_debug64_one(struct qdl_device *qdl,
 	if (!buf)
 		return -1;
 
-	fd = openat(ramdump_dir, region.filename, O_WRONLY | O_CREAT, 0644);
+	char path[PATH_MAX];
+	snprintf(path, sizeof(path), "%s/%s", ramdump_path, region.filename);
```

Comment:
`PATH_MAX` can be somewhat irrelevant for the actual platforms. What about simply allocating a buffer and doing strcpy to it?

Reply:
I was using the same pattern as https://github.com/linux-msm/qdl/blob/master/program.c#L177 and https://github.com/linux-msm/qdl/blob/master/read.c#L112. I'm happy to change it to a buffer allocation if you prefer.